### ISSUE TITLE
Check Simplified Chinese localization

### ISSUE THREAD

**Issue by birtles** [birchill/10ten-ja-reader#683]
Hi @SaltfishAmi, would you have a chance to check the recent changes to the Chinese localization?

The changes are as follows:

```diff
diff --git a/_locales/zh_hans/messages.json b/_locales/zh_hans/messages.json
index 6f4c802..42709af 100644
--- a/_locales/zh_hans/messages.json
+++ b/_locales/zh_hans/messages.json
@@ -71,6 +71,22 @@
     "message": "词语",
     "description": "Label for key to copy a word to the clipboard"
   },
+  "content_database_updating": {
+    "message": "由于字典数据正在更新，所以只显示了一些数据。",
+    "description": "Status bar message shown while the is being updated and only a snapshot can be shown."
+  },
+  "content_hint_switch_dict_keys_1": {
+    "message": "提示：您可以按 %KEY% 查看下一个词典",
+    "description": "Message displayed at the bottom of the pop-up instructing the user which keys to use to switch keyboards when there is only one such key configured"
+  },
+  "content_hint_switch_dict_keys_2": {
+    "message": "提示：您可以按 %KEY% 或 %KEY% 键查看下一个字典",
+    "description": "Message displayed at the bottom of the pop-up instructing the user which keys to use to switch keyboards when there are two keys configured"
+  },
+  "content_hint_switch_dict_keys_3": {
+    "message": "提示：您可以通过按 %KEY%，%KEY% 或 %KEY% 来显示以下词典",
+    "description": "Message displayed at the bottom of the pop-up instructing the user which keys to use to switch keyboards when there are three keys configured"
+  },
   "content_kanji_base_radical": {
     "message": "从​$char$​（​$readings$​）",
     "description": "The string used to describe the base radical",
@@ -159,10 +175,6 @@
     "message": "1画",
     "description": "The description of the number of strokes when there is exactly 1 stroke"
   },
-  "content_names_dictionary": {
-    "message": "名字词典",
-    "description": "The title used at the top of the popup when showing the names dictionary"
-  },
   "content_names_tag_char": {
     "message": "角色"
   },
@@ -425,14 +437,6 @@
   "error_command_too_many_modifiers": {
     "message": "过多的修饰键"
   },
-  "error_ctrl_alt_unsupported": {
-    "message": "当前版本的 Firefox 不支持设置同时按下 Alt 和 Ctrl 的组合键",
-    "description": "Tooltip displayed when the user attempts to set a hotkey combination including both Ctrl and Alt in a version of Firefox prior to 63"
-  },
-  "error_ctrl_alt_warning": {
-    "message": "较早版本（低于 63）的 Firefox 不支持设置同时按下 Alt 和 Ctrl 的组合键",
-    "description": "Tooltip displayed when the user attempts to set a hotkey combination including both Ctrl and Alt in a version of Firefox later than 63"
-  },
   "error_loading_dictionary": {
     "message": "词典加载失败，请重试。",
     "description": "Tooltip shown when the dictionary fails to load"
@@ -1065,6 +1069,22 @@
     "message": "zh-Hans",
     "description": "The BCP-47 compatible language code used to tag content from this localization"
   },
+  "measure_expl_chuukyouma": {
+    "message": "（东海）",
+    "description": "Parenthetical explanation used when presenting alternative conversions between 畳 and square metres"
+  },
+  "measure_expl_danchima": {
+    "message": "（住宅小区）",
+    "description": "Parenthetical explanation used when presenting alternative conversions between 畳 and square metres"
+  },
+  "measure_expl_edoma": {
+    "message": "（东日本）",
+    "description": "Parenthetical explanation used when presenting alternative conversions between 畳 and square metres"
+  },
+  "measure_expl_kyouma": {
+    "message": "（西日本）",
+    "description": "Parenthetical explanation used when presenting alternative conversions between 畳 and square metres"
+  },
   "menu_enable_extension": {
     "message": "启用 10ten reader",
     "description": "Context menu item to enable the extension."
@@ -1310,11 +1330,11 @@
     "description": "Descriptive text used for other browsers which do not permit configuring extension keys from the extension itself."
   },
   "options_browser_commands_no_toggle_key_chrome": {
-    "message": "您无法在浏览器中在此处更改此快捷键。 请访问chrome://extensions/shortcuts并更改设置。",
+    "message": "如果要更改此快捷键，请访问chrome://extensions/shortcuts并更改设置。",
     "description": "Descriptive text used for Chrome which does not permit configuring extension keys from the extension itself."
   },
   "options_browser_commands_no_toggle_key_edge": {
-    "message": "您无法在浏览器中在此处更改此快捷键。 请访问edge://extensions/shortcuts并更改设置。",
+    "message": "如果要更改此快捷键，请访问edge://extensions/shortcuts并更改设置。",
     "description": "Descriptive text used for Edge which does not permit configuring extension keys from the extension itself."
   },
   "options_bushu_data_name": {
@@ -1540,6 +1560,10 @@
   "options_popup_copy_word_kanji": {
     "message": "复制词语或汉字"
   },
+  "options_popup_kanji_lookup": {
+    "message": "只检查汉字结果",
+    "description": "Label for the key to force looking up the kanji dictionary only"
+  },
   "options_popup_keys_subheading": {
     "message": "窗口控制键"
   },
@@ -1630,6 +1654,10 @@
       }
     }
   },
+  "popup_close_label": {
+    "message": "关闭",
+    "description": "The aria-label associated with the button to close the popup"
+  },
   "pos_label_adj_f": {
     "message": "连体",
     "description": "Short part-of-speech label shown next to noun or verb entries acting prenominally"
@@ -2021,5 +2049,17 @@
   "ref_label_unicode": {
     "message": "Unicode",
     "description": "The label used when presenting the Unicode codepoint of the kanji"
+  },
+  "tabs_kanji_label": {
+    "message": "汉字",
+    "description": "The label for the kanji dictionary tab"
+  },
+  "tabs_names_label": {
+    "message": "名字",
+    "description": "The label for the names dictionary tab"
+  },
+  "tabs_words_label": {
+    "message": "单词",
+    "description": "The label for the words dictionary tab"
   }
}
```

Many of them are just guesses / Google translate automatic translations.

Thank you!

**Comment by SaltfishAmi** [comment]
Of course, but I'll have to do it next week. I hope you don't mind 
waiting for a few days :)


SaltfishAmi

**Comment by birtles** [comment]
> Of course, but I'll have to do it next week. I hope you don't mind waiting for a few days :)

Thank you! Yes, I think that will be fine. At this rate I hope to have everything ready to release by about Tuesday JST but I suppose even if this is the last thing blocking release I can just do a point release later with any fixes.

**Comment by shirakaba** [comment]
A note for the English localisation:

> Status bar message shown while the is being updated

It appears to be missing a word after “the” (unless something gets filled in there afterward?).

**Comment by birtles** [comment]
> A note for the English localisation:
> 
> > Status bar message shown while the is being updated
> 
> It appears to be missing a word after “the” (unless something gets filled in there afterward?).

Thanks! Fixed in c4d0f79c4db495a1573c42b1c2dade69daa6384f

**Comment by birtles** [comment]
Fixed by 893f203a5850fe23b5289d0550143caa3bf12673
